extensions/amp-tiktok/0.1/test/validator-amp-tiktok.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
Please add a copyright statement and a simple description of the test in the HTML comments. See other test cases as an example. You'll also need to rerun the update-tests script in order to pass the tests.

Reply:
Looks like this is required in the extension generator as well: #32733